chess/frontend/gui/src/modules/Game/Game.jsx

```diff
@@ -225,6 +225,32 @@ export default class Game extends React.Component {
     });
   }
 
+  getOpponentMove() {
+    return $.ajax({
+      url: "http://127.0.0.1:8000/game/get_opponent_move/",
+      method: "GET",
+      data: {
+        player_color: this.state.turn
+      }
+    }).then(response => {
+      let newStateSquares = this.state.squares;
+
+      response.moves.forEach(move => {
+        let from_coord_response = move.from_coord
+        let to_coord_response = move.to_coord
+
+        let pieceToMove = this.state.squares[TRANSLATE_POSITION[from_coord_response]];
+        newStateSquares[TRANSLATE_POSITION[to_coord_response]] = pieceToMove;
+        newStateSquares[TRANSLATE_POSITION[from_coord_response]] = new Empty(null);
```

Comment:
Turn this into a separate function to be used in both `makeMove` and `getOpponentMove`